Detectors/ZDC/simulation/src/Digits2Raw.cxx

```diff
@@ -412,16 +325,16 @@ void Digits2Raw::insertLastBunch(int ibc, uint32_t orbit)
   assignTriggerBits(ibc, bc, orbit, true);
 
   // Insert payload for all channels
-  for (Int_t im = 0; im < NModules; im++) {
-    for (UInt_t ic = 0; ic < NChPerModule; ic++) {
+  for (int32_t im = 0; im < NModules; im++) {
+    for (uint32_t ic = 0; ic < NChPerModule; ic++) {
       if (mModuleConfig->modules[im].readChannel[ic]) {
         auto id = mModuleConfig->modules[im].channelID[ic];
         auto base_m = mSimCondition->channels[id].pedestal;      // Average pedestal
         auto base_s = mSimCondition->channels[id].pedestalFluct; // Baseline oscillations
         auto base_n = mSimCondition->channels[id].pedestalNoise; // Electronic noise
-        Double_t base = gRandom->Gaus(base_m, base_s);
-        Int_t is = 0;
-        Double_t val = base + gRandom->Gaus(0, base_n);
+        double base = gRandom->Gaus(base_m, base_s);
+        int32_t is = 0;
```

Comment:
again unused `is` which is then incremented

Reply:
It's a leftover from previous code. Thanks for spotting this.